helidon-io/helidon
# Uplevel Nima

Uplevel Nima

Fixes #7164
Fixes #7339
Fixes #7375

```
/nima/http -> /http
/nima/http2 -> /http/http2
/common/http -> /http/http
/nima/webclient -> /webclient
/nima/webserver -> /webserver
/nima/websocket/websocket -> /websocket
/nima/websocket/webserver -> /webserver/websocket
/nima/websocket/webclient -> /webclient/websocket
/nima/sse -> /http/sse
/nima/observe -> /webserver/observe
/nima/test -> /webserver/testing
/nima/fault-tolerance -> /fault-tolerance
/nima/service-common -> /webserver/service-common
/nima/graphql/server -> /webserver/graphql
/nima/common/tls -> /common/tls
/security/integration/nima -> /webserver/security
/nima/tests/webserver -> /webserver/tests
/nima/tests/websocket/server -> /webserver/tests/websocket
/nima/tests/http2/server -> /webserver/tests/http2
/nima/tests/observe -> /webserver/tests/observe
/nima/tests/grpc -> /webserver/tests/grpc
/nima/tests/sse/server -> /webserver/tests/sse
/tests/integration/webserver/* -> /webserver/tests/
/nima/tests/webclient -> /webclient/tests
/nima/tests/http2/client -> /webclient/tests/http2
/tests/integration/webclient -> /webclient/tests/
/nima/tests/encoding -> /http/tests/encoding
/nima/tests/logging/log4j -> /logging/tests/log4j
/nima/tests/media -> /http/tests/media
```

- Remove leftover occurrences of Nima, Níma, nima, NIMA
- Fix ModuleInfoDescriptorBlueprint.mergeCreate
- Fix WebServer feature definition
- Fix Neo4J feature definition
- Update keystores in webserver/tests/mtls (to get rid of 'nima' references)
